### ISSUE TITLE
p:layout in custom dialog make all content invisible on PF elite versions

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#2653]
Reported By PRO User;
> While using primefaces-6.1.4 we have located a regression.
When combining <p:layout> within a dialog, it opened almost empty (see emptyDialog.jpg) that means that all our components are missing - seems that the show method on the dialog is failing so while page DOM contains the expected html, it is not initialized. 

Test code;
```
<h:form id="form">
    <h:panelGrid columns="1" cellpadding="5">
        <p:commandButton value="Basic" type="button" onclick="PF('dlg1').show();" />
    </h:panelGrid>
    <p:dialog header="Basic Dialog" widgetVar="dlg1" minHeight="40">
        <p:layout style="min-width:400px;min-height:200px;">
            <p:layoutUnit position="west" resizable="true" minSize="40" maxSize="400">
                West
                <p:inputText />
            </p:layoutUnit>

            <p:layoutUnit position="center">
                Center
                <p:commandButton value="Test"/>
            </p:layoutUnit>
        </p:layout>
    </p:dialog>
</h:form>
```
When we used primefaces-6.1.jar or 6.1.3, everything worked just fine.

**Comment by PlassJ** [comment]
Got the same Problem in 6.1.4 while using p:layout inside a tabView.
I have read #2395 but didn`t got the point. 
Will this be fixed in 6.1.5 ?

**Comment by melloware** [comment]
@PlassJ Yes this will be fixed in 6.1.5.  Mert and Thomas are creating a better fix and putting that fix in all components that need it like p:layout.

**Comment by PlassJ** [comment]
Thank you
